src/test/euler/staking/StakeableEulerERC4626.t.sol

```diff
@@ -79,24 +79,53 @@ contract StakeableEulerERC4626Test is Test {
         vault.stake(1000);
     }
 
-    function testStaking() public {
-        _setStakingContract();
+    function testStaking() public withStakingContract {
         uint256 deposited = 10000;
         uint256 staked = 1000;
 
-        vm.prank(alice);
-        underlying.approve(address(vault), deposited);
-        vm.prank(alice);
-        vault.deposit(deposited, alice);
-
-        vm.prank(owner);
-        vault.stake(staked);
+        _deposit(alice, deposited);
+        _stake(staked);
 
         assertEq(eToken.balanceOf(address(vault)), deposited - staked);
         assertEq(eToken.balanceOf(address(stakingRewards)), staked);
         assertEq(stakingRewards.balanceOf(address(vault)), staked);
     }
 
+    function testFailNotOwnerUnstaking() public {
+        vault.unstake(1000);
+    }
```

Comment:
How do we know is testing `NotOwner` and not another error? 😢

Reply:
Added it in https://github.com/Mean-Finance/yield-daddy/pull/13